lavamoat/browserify/beta/policy.json

```diff
@@ -3728,6 +3728,12 @@
         "@babel/runtime": true
       }
     },
+    "dompurify": {
+      "globals": {
+        "console.warn": true,
+        "define": true
```

Comment:
I'm not sure what `define` is, can someone give me pointers on what this is?

Reply:
Haven't checked policies in details; were they all generated by metamaskBot or did you have to generate them locally?

Reply:
Good point. I generated them locally, and will double check metamask bot tomorrow.

Reply:
Metamask-bot does not generate any new policy changes, so this is valid.

I'm not sure when `define` policy maps to... Maybe Object.defineProperty()? But I did not see this in the library source code

Reply:
More precisely, this is the UMD module (U for universal) that detects what kind of pre-esm modularization is present. Tries CJS, AMD and then falls back to setting a global. Since we're bundling with CJS, none of the rest is reached.